setup.cfg

```diff
@@ -43,6 +43,7 @@ dev =
     twine
     setuptools_scm[toml]>=3.4.3
     black==19.10b0
+    isort==5.9.1
```

Comment:
any reason we're pinning isort?

Reply:
I just pinned it because black was pinned, as well. Besides that there is no particular reason. Do you want me to unpin it?

Reply:
black is pinned for some bug that makes the formatting different and will cause issues if it's not that version. However, could probably be changed as well at some point to something `>=` once we identify the version thats borked.

Reply:
black is also only in beta-4, so it's probably best to pin it until its stable.

Reply:
unpin isort